python/src/TensorProductExperiment_doc.i.in

```diff
@@ -0,0 +1,117 @@
+%feature("docstring") OT::TensorProductExperiment
+"Gauss product experiment.
+
+Available constructors:
+  TensorProductExperiment(*experiments*)
```

Comment:
no need to list constructors when there is only one

Reply:
Ok, I remove it for consistency with other pages (but I still feel that this would be useful).